packages/browser/__tests__/login/oidc/redirectHandler/GeneralRedirectHandler.spec.ts

```diff
@@ -28,8 +28,9 @@ import {
 import GeneralRedirectHandler from "../../../../src/login/oidc/redirectHandler/GeneralRedirectHandler";
 import {
   generateJWK,
-  signJWT,
+  // signJWT,
```

Comment:
```suggestion
```

Reply:
Removed in 6176710526f4f13537f4419ec557ba6bd2a9e849